src/main.rs

```diff
@@ -133,20 +133,20 @@ fn run() -> i32 {
         },
     };
 
+    let screen_rects: Vec<util::Rect> = match display.get_screen_rects(root) {
+        Some(r) => r.map(|r| r.clone()).collect(),
+        None => {
+            eprintln!("Failed to get screen rects");
+            return 1;
```

Comment:
This means that shotgun will always bail out if screens can't be enumerated, even when it's not strictly required. XRandR should be available on pretty much anything these days, do we care enough to support the failure case?